### PR TITLE
Fixing some PRO tags

### PR DESCRIPTION
**Pull request by jobisoft**

### CODE REVIEW

**Comment by jobisoft** [comment]
I think the Pro version is fixed now as well and I prepared everything for the "switch". If this PR gets accepted, I will send this version to AMO for review. 

The "Pro" branch should be set as default, after this PR has been accepted.

File: chrome/content/quicktext.js [hunk 1]
```diff
@@ -367,36 +367,24 @@ var quicktext = {
   {
     if (aStr != "")
     {
-	  aStr = gQuicktextVar.parse(aStr);
+      aStr = gQuicktextVar.parse(aStr);
       var files = aStr.split(";");
+      var ioService = Components.classes["@mozilla.org/network/io-service;1"].getService(Components.interfaces.nsIIOService);
+      var fileHandler = ioService.getProtocolHandler("file").QueryInterface(Components.interfaces.nsIFileProtocolHandler);
```
Comment (lieser) [inline chrome/content/quicktext.js]: Is `fileHandler` (an wherefore also `ioService`) used anywhere?

**Review by lieser — CHANGES_REQUESTED** [review]
Nice to see that you got also the pro version to run again.

Changes looks good, just 2 unused variables I think.

**Comment by jobisoft** [comment]
Removed. They are not used anywhere else.

**Review by lieser — APPROVED** [review]

**Comment by lieser** [comment]
Thanks again for you effort.

I think it would be good to start tagging the releases, So please let me now from which commit you will do the release (i assume it will be the 5ab56cdbd0764d7c28e216b199d241bf868fed3d, just want to make sure)

**Comment by jobisoft** [comment]
I usually keep the github releases in sync with the AMO releases and I think the AMO review will ask for more changes. So the current 0.9.14 will not be final. My plan is to fix the AMO issues and than release v1.0 - which can also be tagged/released on github. 

Would you like to tag/release 0.9.12? The current official version?

If you ask me, I would start tagging with v1.0 ...

**Pull request merged**